plugins/webp-uploads/uninstall.php

```diff
@@ -2,8 +2,8 @@
 /**
  * Plugin uninstaller logic.
  *
- * @package performance-lab
- * @since 1.2.0
+ * @package webp-uploads
+ * @since n.e.x.t
```

Comment:
The `n.e.x.t` replacements script at the moment still only works for the main PL plugin, so for the other plugins so far we have to handle it ourselves until we have improved the script.
```suggestion
 * @since 1.1.0
```